### PR TITLE
docs: document Scala Native flag options

### PR DESCRIPTION
**Pull request by scarf005**
referenced from: https://github.com/scala-native/scala-native/blob/157355f4bcbb4a2873c0c7ccdc9be4be969135e2/docs/user/sbt.md?plain=1#L132

### CODE REVIEW

File: website/docs/reference/scala-command/runner-specification.md [hunk 1]
```diff
@@ -182,15 +182,15 @@ Set the Scala Native version (0.5.6 by default).
 
 **--native-mode**
 
-Set Scala Native compilation mode
```
Comment (Gedochao) [inline website/docs/reference/scala-command/runner-specification.md:185]: Everything under `website/docs/reference` is generated automatically with
```bash
./mill -i 'generate-reference-doc[]'.run
```

This needs to be adjusted in the Scala Native options implementation first:
https://github.com/VirtusLab/scala-cli/blob/4ab322768f7b5813312aeb553fed8a2caf35e4a4/modules/cli/src/main/scala/scala/cli/commands/shared/ScalaNativeOptions.scala#L22

**Review by Gedochao — CHANGES_REQUESTED** [review]

File: website/docs/reference/scala-command/runner-specification.md (hunk 1 above)
Comment (scarf005) [reply website/docs/reference/scala-command/runner-specification.md:185]: applied and force-pushed in 29f790e0546f2ee98ff4da92741672de6f322c5a.

**Review by Gedochao — APPROVED** [review]
LGTM, thanks for the contribution!

**Comment by He-Pin** [comment]
Thanks

**Pull request merged**